basic/exercise1/script.js

```diff
@@ -9,10 +10,12 @@ function formatDate(date) {
     var hours = date.getHours();
     var ampm = hours >= 12 ? 'PM' : 'AM';
     hours = hours % 12;
+    var formatHours = formatAsTwoDigits(hours);
```

Comment:
You have some inconsistency in variables names:
- formatted hours named as formatHours
- formatted minutes and seconds named as minutes and seconds.
Please unify it.